### PR TITLE
[Bulk] Validation - Duplicate alt tags

### PR DESCRIPTION
**Pull request by vatmicro**
This PR fixes 28 duplicate alt tag issues

### CODE REVIEW

File: api/Word.Options.MatchFuzzyBV.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ _expression_. `MatchFuzzyBV`
...
-![Symbol](../images/fe268_ZA06051747.gif)" and between "
-![Symbol](../images/fe278_ZA06051757.gif)" and "
-![Symbol](../images/fe238_ZA06051718.gif)
-![Symbol](../images/fe268_ZA06051747.gif)" during a search.
+![A screenshot that shows the first symbol in the example.](../images/fe143_ZA06051648.gif)" and "
+![A screenshot that shows the second symbol in the example.](../images/fe267_ZA06051746.gif)
+![A screenshot that shows the third symbol in the example.](../images/fe268_ZA06051747.gif)" and between "
+![A screenshot that shows the fourth symbol in the example.](../images/fe278_ZA06051757.gif)" and "
+![A screenshot that shows the fifth symbol in the example.](../images/fe238_ZA06051718.gif)
+![A screenshot that shows the sizth symbol in the example.](../images/fe268_ZA06051747.gif)" during a search.
```
Comment (jmart1428) [inline api/Word.Options.MatchFuzzyBV.md:41]: typo on 'sixth'

**Comment by vatmicro** [comment]
Global effort to fix validation errors

@lindalu-MSFT 

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]
Please change "A screenshot that shows ___" to "Screenshot of"

**Comment by vatmicro** [comment]
Hi @lindalu-MSFT , I have made the changes that you requested.

**Review by lindalu-MSFT — APPROVED** [review]

**Pull request merged**